osbs/core.py

```diff
@@ -45,6 +45,12 @@
 WAIT_RETRY_HOURS = 12
 WAIT_RETRY = WAIT_RETRY_HOURS * 3600 // (WATCH_RETRY_SECS * WATCH_RETRY)
 
+OCP_RESOURCE_API_VERSION_MAP = {
+    'builds': 'build.openshift.io/v1',
+    'buildconfigs': 'build.openshift.io/v1',
+    'imagestreams': 'image.openshift.io/v1',
```

Comment:
Maybe we should give symbolic names to these strings?
```
OCP_RESOURCE_API_VERSION_BUILD = 'build.openshift.io/v1'
OCP_RESOURCE_API_VERSION_IMAGE = 'image.openshift.io/v1'
```
to prevent typos and to put the actual versions we're using in a central location.

Reply:
I ahve it in core.py, shall I move it to constants.py? (It's not used elsewhere)